### PR TITLE
Removes bootstrap, adds initial base styles and homepage design

### PR DESCRIPTION
**Pull request by smharley**
image

Comments:
- I got rid of bootstrap, so the rest of the site is going to look terrible. I tried adding it back, and it kind of messed up what I was working on. I'm starting a new client project tomorrow, so I won't be able to finish the rest of the front-end in the very near term, which makes me nervous breaking all the other pages. Any thoughts?
- I made up all the words that are in there, we might want to put some additional thought into them before a launch
- I added the sign up button directly to the home page, so when you click it, you're given this awkward redirect to the homepage again. I'd like this to redirect to the `sites` index

Questions:
- Can we add bootstrap just to pages that aren't the homepage? Should we just add it back and deal with the homepage not being perfect for a bit?
- Not really sure how to split up the CSS. Ideally there would be some folders and organization via `@imports` like SASS. It's not unmanageable yet, but it will grow a lot

### CODE REVIEW

**Comment by pbrisbin** [comment]
> I added the sign up button directly to the home page, so when you click it, you're given this awkward redirect to the homepage again. I'd like this to redirect to the sites index

I can take care of this.

> Can we add bootstrap just to pages that aren't the homepage? 

Yes, if it comes to that, that can easily be done.

> Not really sure how to split up the CSS. Ideally there would be some folders and organization via @imports like SASS.

This is something we're going to have to figure out. Yesod favors heavily towards the concept of "widgets" -- isolated units of HTML/CSS/JavaScript. Under this scheme, we'd define a `navWidget = $(widgetFile "nav")`, `footerWidget = $(widgetFile "footer")`, `heroWidget = $(widgetFile "hero")`, then you'd get organization for free because these widgets are built from `nav.lucius`, `footer.lucius`, and `hero.lucuis`. What I'm not sure of is how best to handle cross-cutting concerns.

Anyway, this is something I can take on myself.

> I got rid of bootstrap, so the rest of the site is going to look terrible. I tried adding it back, and it kind of messed up what I was working on. I'm starting a new client project tomorrow, so I won't be able to finish the rest of the front-end in the very near term, which makes me nervous breaking all the other pages. Any thoughts?

I'll pick up this branch and see what I can do to complete it enough to merge and deploy. Would you be available on Friday to check back in and smooth over any rough edges I create?

**Comment by smharley** [comment]
@pbrisbin that all sounds good! the widgets thing will work for most things, that normally how I write my styles, but I think you're right, there are some more 'global' styles that might need a better home. Today was my investment day, so I won't have a full day on Friday but will be able to smooth things over if necessary

File: templates/root.lucius [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  padding: 0;
+}
+
+.button {
+  background: #{linkcolor};
+  border-radius: 1.125em;
+  color: #fff;
+  display: inline-block;
+  line-height: 1;
+  padding: 8px 1.5em 9px;
```
Comment (donokuda) [inline templates/root.lucius]: I wonder why the padding on the top and bottom are off.. Is it a button thing?

**Comment by donokuda** [comment]
@smharley One comment LGTM :+1:

File: templates/root.lucius (hunk 1 above)
Comment (smharley) [inline templates/root.lucius]: I think it's how the font is getting rendered, the line-height is just not centered. it's just an `<a>` so probably not a button thing.

**Comment by pbrisbin** [comment]
@smharley FYI: I'm picking up work on this branch and I'm a frequent rebaser. Give me a heads-up if you start any work on this again and I'll hold off on rewriting history.

**Comment by pbrisbin** [comment]
Took care of making the other pages more palatable by bringing back (a customized subset of) bootstrap. I made some additional style changes probably best reviewed in their individual commits. Let me know if this is good enough to deploy or I should wait for you to make adjustments.



@smharley do you notice how the logo looks for me -- I'm not sure why...

**Comment by pbrisbin** [comment]
I'm happy enough to merge this, but the logo is a blocker.

**Comment by smharley** [comment]
The logo is strange.. I can just make it a .png. I'll try to do that later today

**Comment by pbrisbin** [comment]
So, I just used this online converter thing to make a PNG from the SVG and got this:

logo

So it seems something's really wrong, it's not just a Linux problem!

**Comment by pbrisbin** [comment]
Woot, I was able to make PNG in GIMP and it came out correct (as far as I can tell).

**Comment by pbrisbin** [comment]
@smharley would you be able to pull this down and take a peak at all the pages today? With your thumbs-up, I'd like to merge and deploy ASAP.

File: templates/root.lucius [hunk 2]
```diff
@@ -0,0 +1,28 @@
+@linkcolor: #E74C3D;
+@linkbordercolor: #F7C4BF;
+@lightgray: #D7D7D7;
+@mediumgray: #615E5F;
+@darkgray: #221E1F;
```
Comment (pbrisbin) [inline templates/root.lucius]: Right now I've just duplicated these variables in the lucius files that need (any of) them. I've opened https://github.com/yesodweb/shakespeare/issues/152 to look into the best way to reduce that duplication.

**Comment by pbrisbin** [comment]
I'm going to go ahead and merge this. There was already a designer +1, it's a huge improvement over what's out there now, and I've been heavily testing it locally.

**Comment by pbrisbin** [comment]
@smharley I'll leave this branch up in case you want it for anything. Please clean it up if you don't.

**Pull request merged**